README.md

```diff
@@ -29,8 +29,8 @@ repository: https://github.com/the/address/to/the/gem.git
 
 When creating a new GEM please consider:
 
-* the name should express the functionality of the GEM clearly
-* words should be separated by using dash (-)
-* don't use *under_score* and *CamelCase*
-* prefix should be *mruby-*
-* if it's an executable, prefix should be *mruby-bin-*
+- the name should express the functionality of the GEM clearly
+- words should be separated by using dash (-)
+- don't use _under_score_ and _CamelCase_ <!-- markdownlint-disable MD037 -->
```

Comment:
I hate this kind of control comment to silence the checker/linter. Is there any other way to address this?

Reply:
Yes: surround the word [with asterisks](https://github.github.com/gfm/#emphasis-and-strong-emphasis) instead:

```diff
diff --git a/README.md b/README.md
index 0c60cb1..e0b6b6a 100644
--- a/README.md
+++ b/README.md
@@ -31,6 +31,6 @@ When creating a new GEM please consider:
 
 - the name should express the functionality of the GEM clearly
 - words should be separated by using dash (-)
-- don't use _under_score_ and _CamelCase_ <!-- markdownlint-disable MD037 -->
+- don't use *under_score* and _CamelCase_
 - prefix should be _mruby-_
 - if it's an executable, prefix should be _mruby-bin-_
```

Reply:
@rdipardo Great! Thank you!